kubernetes-config/loadbalancer.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: github-nginx-cache-loadbalancer
+spec:
+  ports:
+    - port: 80
+      protocol: TCP
+      targetPort: 80
+  selector:
+    app: github-nginx-cache
+  type: LoadBalancer
```

Comment:
these are just samples i presume, the actual specs will be in the deployment repo   ( also why do we need type loadBalancer ) ?

Reply:
yeah (Y)

Reply:
Loadbalancer was just for testing